src/coredata.cs

```diff
@@ -387,6 +428,11 @@ public interface NSFetchRequest : NSCoding {
 		[Export ("propertiesToGroupBy", ArgumentSemantic.Copy)]
 		[NullAllowed]
 		NSPropertyDescription [] PropertiesToGroupBy { get; set; }
+
+		[Watch (3,0), TV (10,0), iOS (10,0)]
+		[Export ("execute:")]
+		[return: NullAllowed]
+		NSFetchRequestResult[] Execute (out NSError error);
```

Comment:
should be `INSFetchRequestResult` since there's no concrete type (once BaseType is removed above it should warn at build time).